### DISCUSSION TITLE
[Feature Req] Android Monochrome/Material You Icon

### DISCUSSION THREAD

**Discussion by namePlayer** [astubenbord/paperless-mobile#199] [Feature Requests]
Story: I have enabled Material You based Colored Icons. As the overall look of the home screen get a lot cleaner with all icons in the same colorway, it would be great to also see such an icon for the Paperless Mobile App.

On the technical side it should be really easy to solve, but I don't guarantee because I don't have lots of experience with Android App Development.
I got two links from the android documentation how to implement themed icons:
- https://developer.android.com/develop/ui/views/launch/icon_design_adaptive
- https://developer.android.com/studio/write/create-app-icons#create-adaptive

Thanks in advance!

**Comment by astubenbord** [comment]
Done, will be updated soon :)

**Reply by namePlayer** [reply]
Thanks! Appreciate it 😃

**Reply by astubenbord** [reply]
Sure, it's in the Open Beta for now, If it works for you I'll publish it for everyone :)
